exalearn-ui/src/app/components/new-grammar/new-grammar.component.html

```diff
@@ -23,13 +23,24 @@
 			</mat-form-field>
 		</div>
 
+		<div class="reminder-answer">
+			<div *ngIf="!rightAnswer" class="reminder-text">
+				<p>Don't forget about right answer</p>
+			</div>
+			<div *ngIf="rightAnswer" class="reminder-error">
+				<p>Please select the right answer</p>
+			</div>
+			<div class="reminder-pointer">
+				<mat-icon color="primary">arrow_downward</mat-icon>
+			</div>
+		</div>
+
 		<div formArrayName="answers">
 			<div *ngFor="let item of answers['controls']; let i = index" [formGroupName]="i" class="answer-container">
 				<mat-form-field appearance="fill">
 					<mat-label>Answer {{i+1}}</mat-label>
 					<input matInput type="text" formControlName="text" />
 					<mat-error *ngIf="item.get('text')?.hasError('required')">Required</mat-error>
-					<mat-error *ngIf="item.get('text')?.hasError('minlength')">Min 2 symbols</mat-error>
```

Comment:
As I remember we discussed about 1 required symbol instead 2

Reply:
yeah. We really had discussion about it. But here are  "required" and "empty string" validation. And if somebody inputs space, space or space  - the field turns to invalid. Well, if we have "required" and "empty string" validation, we needn't use 1 required symbol validation 👀